ndex2/client.py

```diff
@@ -653,8 +852,25 @@ def get_network_summary(self, network_id):
 
     def make_network_public(self, network_id):
         """
+
         Makes the network specified by the network_id public.
 
+        .. code-block:: python
+
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+
+		network_id = "INSERT NETWORK UUID HERE"
+		my_ndex.make_network_public("network_id")
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client